projects.go

```diff
@@ -75,6 +75,8 @@ type Project struct {
 	EmptyRepo                                 bool                       `json:"empty_repo"`
 	Archived                                  bool                       `json:"archived"`
 	AvatarURL                                 string                     `json:"avatar_url"`
+	License                                   *ProjectLicense            `json:"license"`
+	LicenseURL                                string                     `json:"license_url"`
```

Comment:
Please switch these two fields so they follow the order in the docs...

Reply:
I've switched the fields as you requested, however note that there are some existing fields that are not in order compared to the docs. See for example `MarkedForDeletionAt` and `EmptyRepo`.

Reply:
Check. Thanks for the heads up... Probably missed that in an earlier PR...